components/automate-ui/src/app/pages/notifications/notifications.component.ts

```diff
@@ -70,6 +83,15 @@ export class NotificationsComponent implements OnInit {
     this.toggleSort('name');
   }
 
+  ngOnDestroy(): void {
+    this.isDestroyed.next(true);
```

Comment:
This is not hooked up--need to add `takeUntil(this.isDestroyed)` on subscriptions in this file.